src/cards.json

```diff
@@ -115,4 +115,5 @@
   { "title": "戒色第100天", "type": "trap", "src":"https://imgur.dcard.tw/trx1tL8h.png"},
   { "title": "眾基首", "type": "monster", "src":"https://i.imgur.com/VieLwoz.jpg"},
   { "title": "斯拉夫之基", "type": "monster", "src":"https://i.imgur.com/iIQTj4s.jpg"}
+  { "title": "防色色陷阱", "type": "trap", "src":"https://imgur.com/a/SANLsVH"}
```

Comment:
Please use direct link instead of short link.

```diff
- { "title": "防色色陷阱", "type": "trap", "src":"https://imgur.com/a/SANLsVH"}
+ { "title": "防色色陷阱", "type": "trap", "src":"https://i.imgur.com/LXSoJ7s.jpeg"}
```